setup.py

```diff
@@ -100,24 +137,34 @@ class DevelopMode(develop):
     def run(self):
         """Install the package in a developer mode."""
         super().run()
-        for _file in ETC_FILES:
-            self.create_path(_file)
 
-    @staticmethod
-    def create_path(file_name):
-        """Method used to create the configurations files using develop."""
-        etc_kytos_dir = os.path.join(BASE_ENV, 'etc/kytos')
+        self.create_paths()
+        self.generate_file_from_template(TEMPLATE_FILES, prefix=BASE_ENV)
+
+        for file_name in ETC_FILES:
+            self.generate_file_link(file_name)
 
+        for file_name in TEMPLATE_FILES:
+            self.generate_file_link(file_name.replace('.template', ''))
+
+    @staticmethod
+    def generate_file_link(file_name):
+        """Method used to create a symbolic link from a file name."""
         current_directory = os.path.dirname(__file__)
         src = os.path.join(os.path.abspath(current_directory), file_name)
         dst = os.path.join(BASE_ENV, file_name)
 
-        if not os.path.exists(etc_kytos_dir):
-            os.makedirs(etc_kytos_dir)
-
         if not os.path.exists(dst):
             os.symlink(src, dst)
 
+    @staticmethod
+    def create_paths():
+        """Method used to create the paths used by Kytos in develop mode."""
+        directories = [os.path.join(BASE_ENV, 'etc/kytos')]
+        for directory in directories:
+            if not os.path.exists(directory):
+                os.makedirs(directory)
+
 
 requirements = [i.strip() for i in open("requirements.txt").readlines()]
 napps_dir = os.path.join(BASE_ENV, 'var/lib/kytos/napps/.installed')
```

Comment:
This information should be provided by the conf file, doesn't it?

I mean, I am not sure if we should handle `napps_dir` during `kytos` setup, since it may be changed based on the kytos.conf file. So, both in kytos and kytos-utils we should check the `napps_dir` when needed and create it if it does not exist, right? @beraldoleal does it make sense? Should this point be addressed on another issu?